pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -384,9 +385,19 @@ var _ = Describe("Converter", func() {
 		var convertedDomain = fmt.Sprintf(`<domain type="%s" xmlns:qemu="http://libvirt.org/schemas/domain/qemu/1.0">
   <name>mynamespace_testvmi</name>
   <memory unit="B">8388608</memory>
-  <os>
+  <os>`, domainType)
```

Comment:
Instead of adding if/else, can you just copy in the whole ppc64 xml and just check against it?

Reply:
Ok, sure will do.